browser-test/login.test.js

```diff
@@ -0,0 +1,71 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {
+  getDriver,
+  getElementByXPath,
+  urls,
+  initialData,
+  executeCommand,
+} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    jest.setTimeout(30000);
```

Comment:
why a timeout of 30 seconds? This operation should not take so long.

Reply:
In the CI, if I used 5 seconds as timeout then it's failing because it takes time in initialization and running-test. Can I make it 15 seconds?